config/database.yml

```diff
@@ -2,13 +2,13 @@ default: &default
   adapter: postgresql
```

Comment:
In order to let users set their prefered config for the DB. I'd suggest actually not tracking this file.